### PR TITLE
CMake configuration file changes to support Windows

### PR DESCRIPTION
**Pull request by yying**
Modified a few `CMakeLists.txt` files to support a Windows build.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by PoppySeedPlehzr** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/2258/merge (Job: 2892) failed :-1:

**Comment by PoppySeedPlehzr** [comment]
ok to test

**Comment by osqueryer** [comment]
Build origin/pr/2258/merge (Job: 2895) failed :-1:

**Comment by osqueryer** [comment]
Build origin/pr/2258/merge (Job: 2896) is good! :+1:

File: third-party [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 2610167fe818c3d67c549185332f193a778f7586
+Subproject commit 1aad215e8bf841bc8ed6321432dd3b1cb0fc7af1
```
Comment (theopolis) [inline third-party]: What is going on here?

Comment (yying) [inline third-party]: Accidental include, going to get rid of it.

**Comment by osqueryer** [comment]
Build origin/pr/2258/merge (Job: 2900) is good! :+1:

File: osquery/logger/CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 
-file(GLOB OSQUERY_LOGGER_PLUGINS "plugins/*.cpp")
-file(GLOB OSQUERY_LOGGER_PLUGIN_TESTS "plugins/tests/*.cpp")
-ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-kinesis")
-ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-firehose")
-ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-core")
+if(WIN32)
+  file(GLOB OSQUERY_LOGGER_PLUGINS "plugins/filesystem.cpp")
+  file(GLOB OSQUERY_LOGGER_PLUGIN_TESTS "plugins/tests/filesystem_logger_tests.cpp")
+ else()
```
Comment (theopolis) [inline osquery/logger/CMakeLists.txt]: Oh no, a wild ` ` has appeared!

**Comment by theopolis** [comment]
One nit and let's merge.

**Comment by osqueryer** [comment]
Build origin/pr/2258/merge (Job: 2908) is good! :+1:

**Pull request merged**